### PR TITLE
adhoc/ivory 556 various additional legal changes

### PR DESCRIPTION
**Pull request by DashCoder**

### CODE REVIEW

File: server/views/what-type-of-item-is-it.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
           }
       }) %}
 
-        <p class="govuk-body" id="introPara">Unless your item is to be sold or hired out to a qualifying museum, any replacement ivory must have been taken from an elephant before 1975 and used only to restore the item.</p>
+        <p class="govuk-body" id="introPara">Unless your item is to be sold or hired out to a qualifying museum, any replacement ivory must have been taken from an elephant before 1975 and added only for the purpose of restoring the item.</p>
```
Comment (duncannixon) [inline server/views/what-type-of-item-is-it.html:24]: Add unit test to check for existence and value of "introPara" (after the unit test "should have the correct page heading")

File: server/views/eligibility-checker/less-than-20-ivory.html [hunk 2]
```diff
@@ -33,8 +33,8 @@
           summaryText: "What kinds of item have less than 20% ivory?",
           html: '<p>Items likely to have less than 20% ivory include:</p>
```
Comment (duncannixon) [inline server/views/eligibility-checker/less-than-20-ivory.html:34]: Add IDs to the P and UL tags and unit test their existence and content

File: server/views/eligibility-checker/contain-elephant-ivory.html [hunk 3]
```diff
@@ -34,7 +34,7 @@
                   <li>fake ivory, such as ivorine</li>
                   <li>certain types of animal bone</li>
                 </ul>
-                <p>If you are not sure about the item you want to sell or buy, you should consult an expert first, such as a registered antiques dealer or auctioneer who specialises in ivory.</p>'
+                <p>If you are not sure about the item you want to sell or buy, you should consult an expert first, such as an antiques dealer or auctioneer who specialises in ivory.</p>'
```
Comment (duncannixon) [inline server/views/eligibility-checker/contain-elephant-ivory.html:37]: Add IDs to the 2 P tags and the UL tag in this HTML block and add unit tests to check for existence and value of those elements.

File: server/views/eligibility-checker/cannot-trade.html [hunk 4]
```diff
@@ -16,6 +16,17 @@ <h1 id="pageTitle" class="govuk-heading-l">{{ pageTitle }}</h1>
 
       <h2 id="heading2" class="govuk-heading-m">What you can do with this item</h2>
 
+
+      <p id="helpText3p">Based on your answers, it is illegal to:</p>
```
Comment (duncannixon) [inline server/views/eligibility-checker/cannot-trade.html:20]: Add unit test for this element (check existence and value). Also I'm not sure about the ID here, would ".....2b' be better (i.e. why does it end in a 'p'?)

File: server/views/eligibility-checker/cannot-trade.html [hunk 5]
```diff
@@ -16,6 +16,17 @@ <h1 id="pageTitle" class="govuk-heading-l">{{ pageTitle }}</h1>
...
+
+      <ul id="helpTextListp" class="govuk-list govuk-list--bullet">
+        <li>sell or hire it out</li>
+        <li>offer or arrange to sell or hire it out</li>
+        <li>keep it for sale or hire</li>
+        <li>import it into the United Kingdom for sale or hire</li>
+        <li>export it from the United Kingdom for sale or hire</li>
+      </ul>
+
```
Comment (duncannixon) [inline server/views/eligibility-checker/cannot-trade.html:29]: Same as above - unit test for this element (check existence and value). Also I'm not sure about the ID here,  remove the 'p' from the end?

**Review by duncannixon — DISMISSED** [review]
The changes seem fine, just need to add/amend unit tests and some suggestions regarding the IDs that are being used. So far so good though :)

**Comment by duncannixon** [comment]
Also - need to increment the build number in package.json

**Review by Jozzey — APPROVED** [review]
Looks good 👍🏼

**Pull request merged**